lib/jazzy/config.rb

```diff
@@ -400,6 +401,12 @@ def print_attr_description(attr)
       end
     end
 
+    class PathnameArray < Array
```

Comment:
where is it that there can either be 1 pathname or an array? That sounds like a bug to me, and one that would be fixed by making sure it's always an array